homeassistant/components/owlet/const.py

```diff
@@ -0,0 +1,11 @@
+"""Constants for the Owlet Smart Sock integration."""
+
+DOMAIN = "owlet"
+
+CONF_OWLET_EXPIRY = "expiry"
+CONF_OWLET_REFRESH = "refresh"
+
+SUPPORTED_VERSIONS = [3]
+POLLING_INTERVAL = 5
+MANUFACTURER = "Owlet Baby Care"
+SLEEP_STATES = {0: "Unknown", 1: "Awake", 8: "Light Sleep", 15: "Deep Sleep"}
```

Comment:
Personally, since these are used in sensors, I may just put it there, but that's not really needed.

You should however make it so you can translate them in strings.json